__tests__/extension.test.js

```diff
@@ -0,0 +1,7 @@
+import { chrome } from "../src";
+
```

Comment:
Are you this is necessary?  I didn't think we needed this import.

Reply:
We don't, must've just slipped in there, I'll take it out.